src/main/java/org/apache/mesos/curator/CuratorConfigStore.java

```diff
@@ -183,6 +182,10 @@ public UUID getTargetConfig() throws ConfigStoreException {
         }
     }
 
+    public void close() {
```

Comment:
Will anyone ever want to call this instead of just destroying the instance?

Maybe make them package-private if they're only for testing